internal/acceptance/template-strings-repository-docker.go

```diff
@@ -24,6 +27,9 @@ resource "nexus_repository_docker_group" "acceptance" {
 {{- end }}
 {{- if .Docker.HTTPSPort }}
 		https_port = "{{ .Docker.HTTPSPort }}"
+{{- end }}
+{{- if .Docker.Subdomain }}
+		https_port = "{{ .Docker.Subdomain }}"
```

Comment:
the tests can not working correct. you overwrite the variable https_port in your test string template instead of adding subdomain
```suggestion
		subdomain = "{{ .Docker.Subdomain }}"
```